src/components/project/Releases.vue

```diff
@@ -0,0 +1,415 @@
+<template>
+  <div class="project-releases">
+    <div
+      v-if="project"
+      class="project-releases__content"
+    >
+      <div
+        v-if="groupedReleases && Object.keys(groupedReleases).length"
+        class="project-releases__list"
+      >
+        <div
+          v-for="(dayReleases, day) in groupedReleases"
+          :key="day"
+          class="project-releases__day"
+        >
+          <div class="project-releases__day-header">{{ formatDayHeader(day) }}</div>
+          <div
+            v-for="(release, index) in dayReleases"
+            :key="release.id"
+            class="project-releases__item"
+            :class="{ 'project-releases__item--expanded': expandedReleases[`${day}-${index}`] }"
+            @click="toggleRelease(`${day}-${index}`)"
+          >
+            <div class="project-releases__item-header">
+              <div class="project-releases__item-info">
+                <div class="project-releases__time">{{ getTimestampFromReleaseName(release.releaseName) | prettyTime }}</div>
```

Comment:
release name is any unique string generated by client. You can't extract date from it. Use "id" instead which includes timestamp